docs/modeling/bounding-boxes.rst

```diff
@@ -0,0 +1,216 @@
+.. _bounding-boxes:
+
+Efficient Model Rendering with Bounding Boxes 
+=============================================
+
+.. versionadded:: 1.1
+
+All `astropy.modeling.Model` subclasses have a ``bounding_box`` attribute that 
+can be used to set the limits over which the model is significant. This greatly 
+improves the effciency of evaluation when the input range is much larger than 
+the characteristic width of the model itself. For example, to create a sky model 
+image from a large survey catalog, each source should only be evaluated over the 
+pixels to which it contributes a significant amount of flux. This task can 
+otherwise be computationally prohibitive on an average CPU. 
+
+The `astropy.modeling.render_model` function can be used to evaluate a model on 
+an input array, or coordinates, limiting the evaluation to the ``bounding_box`` 
+region if it is set.
+
+Using the Bounding Box
+-----------------------
+
+``bounding_box`` is a property that returns a tuple of tuples defining the 
+limits for each input. ``((y_low, y_high), (x_low, x_high), (z_low, z_high))``. 
+Note that the ``'y'``-coordinate 
+comes first for ``model.n_inputs > 1`` following the `numpy` array indexing 
+convention.
+
+By default no bounding box is set (``bounding_box`` is `None`), except for 
+individual model subclasses that have a ``bounding_box_default`` function 
+defined. ``bounding_box_default`` returns the minimum rectangular region 
+symmetric about the position that fully contains the model. If a model is not 
+spatially discrete, e.g. ``Gaussian1D``, the relative error is no more than 
+:math:`10^{-7}`.
```

Comment:
It's not clear to me if 1e-7 only refers to Gauss, of if this is the containment fraction you're using to define the default bounding boxes for all built-in models.
Maybe split this in two sentences. A general one talking about containment fraction and then: "For example, Gaussian1D we have chose 5 sigma as default, so that only `1e-7` of the total flux is outside the bounding box."